### ISSUE TITLE
Got an exception while streaming logs TypeError("ws_connect() got an unexpected keyword argument 'params'")

### ISSUE THREAD

**Issue by mapleeit** [jina-ai/jcloud#7]
My flow.yml

```yaml
jtype: Flow
executors:
- name: sandbox_executor
  uses: 'jinahub+docker://Hello'
```

Everything is fine, I verified that whether the deployed flow works. But when I run `jc logs FLOW_ID`, it throwed error.

<details>

```bash
❯ cat flow.yml
jtype: Flow
executors:
- name: sandbox_executor
  uses: 'jinahub+docker://Hello'

❯ jc deploy flow.yml
/Users/miaozhaofeng/.pyenv/versions/3.9.5/lib/python3.9/site-packages/aiohttp/client.py:977: RuntimeWarning: coroutine 'noop' was never awaited
  self._resp.release()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
[15:02:40] ERROR    Got an exception while streaming logs TypeError("ws_connect() got an unexpected keyword argument 'params'")         flow.py:285
╭────────── 🎉 Flow is available! ──────────╮
│                                           │
│   ID    803b9b5f44                        │
│   URL   grpcs://803b9b5f44.wolf.jina.ai   │
│                                           │
╰───────────────────────────────────────────╯

❯ python client.py
[{"id": "40c740c351e119dc55381d3d51f52608", "parent_id": null, "granularity": null, "adjacency": null, "blob": null, "tensor": null, "mime_type": "text/plain", "text": "hello hello", "weight": null, "uri": null, "tags": null, "offset": null, "location": null, "embedding": null, "modality": null, "evaluations": null, "scores": null, "chunks": null, "matches": null}]

❯ jc logs 803b9b5f44
[15:05:39] ERROR    Got an exception while streaming logs TypeError("ws_connect() got an unexpected keyword argument 'params'")         flow.py:285
```

**Comment by hanxiao** [comment]
`aiohttp` too old?

**Comment by hanxiao** [comment]
fixed in `0.0.8`
